lunatrace/bsl/frontend/src/pages/homepage/cards/SetupWizard.tsx

```diff
@@ -81,9 +81,15 @@ export const SetupWizard: React.FunctionComponent<WizardProps> = ({ compact }) =
 
           <Row className="justify-content-center">
             <Col md={compact ? '12' : '6'} className="d-grid gap-2">
-              <Button variant={hasProjects ? 'light' : 'primary'} size="lg" href={GithubAppUrl}>
-                <BsGithub className="mb-1 me-1" /> {hasProjects ? 'Add more projects' : 'Connect to GitHub'}
-              </Button>
+              {hasProjects ? (
+                <Button variant="light" size="lg" onClick={() => navigate('/project/import')}>
+                  <BsGithub className="mb-1 me-1" /> {'Add more projects'}
+                </Button>
+              ) : (
+                <Button variant="primary" size="lg" href={GithubAppUrl}>
```

Comment:
send new users straight to github since they dont have any projects to manage anyway